pkg/cmd/roachtest/cluster.go

```diff
@@ -3182,23 +3196,27 @@ func (c *clusterImpl) UpdateSideEyeEnvironmentName(
 // swallowed.
 //
 // Returns the URL of the captured snapshot, or "" if not successful.
-func (c *clusterImpl) CaptureSideEyeSnapshot(
-	ctx context.Context, l *logger.Logger, client *sideeyeclient.SideEyeClient,
-) string {
+func (c *clusterImpl) CaptureSideEyeSnapshot(ctx context.Context) string {
+	l := c.t.L()
 	l.PrintfCtx(ctx, "capturing snapshot of the cluster with Side-Eye...")
 
 	if c.arch == vm.ArchARM64 {
 		l.Printf("Side-Eye does not support ARM64 machines; skipping snapshot")
 		return ""
 	}
 
+	if c.sideEyeClient == nil {
+		l.Printf("Side-Eye client is not configured")
```

Comment:
Nit: granted, it's a poorman's logger which doesn't understand levels... we do have a tacit convention to prefix warning messages with `WARN: `.